container-images/kolla/wsgi/cinder-wsgi

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/python3 -sP
+#PBR Generated from 'wsgi_scripts'
+
+import threading
+
+from cinder.wsgi.wsgi import initialize_application
+
+if __name__ == "__main__":
+    import argparse
+    import socket
+    import sys
+    import wsgiref.simple_server as wss
+
+    parser = argparse.ArgumentParser(
+        description=initialize_application.__doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        usage='%(prog)s [-h] [--port PORT] [--host IP] -- [passed options]')
+    parser.add_argument('--port', '-p', type=int, default=8000,
+                        help='TCP port to listen on')
+    parser.add_argument('--host', '-b', default='',
+                        help='IP to bind the server to')
+    parser.add_argument('args',
+                        nargs=argparse.REMAINDER,
+                        metavar='-- [passed options]',
+                        help="'--' is the separator of the arguments used "
+                        "to start the WSGI server and the arguments passed "
+                        "to the WSGI application.")
+    args = parser.parse_args()
+    if args.args:
+        if args.args[0] == '--':
+            args.args.pop(0)
+        else:
+            parser.error("unrecognized arguments: %s" % ' '.join(args.args))
+    sys.argv[1:] = args.args
+    server = wss.make_server(args.host, args.port, initialize_application())
+
+    print("*" * 80)
+    print("STARTING test server cinder.wsgi.wsgi.initialize_application")
+    url = "http://%s:%d/" % (server.server_name, server.server_port)
+    print("Available at %s" % url)
+    print("DANGER! For testing only, do not use in production")
+    print("*" * 80)
+    sys.stdout.flush()
+
+    server.serve_forever()
+else:
```

Comment:
You can remove all of this. This starts a test server which - per the warning - is not to be used in production. I don't see any reason to support it.